### PR TITLE
Add SmallHandleSearch component and its styles

### PR DESCRIPTION
**Pull request by Dauntless-Jacobson**

### CODE REVIEW

**Comment by Dauntless-Jacobson** [comment]
search

**Comment by Dauntless-Jacobson** [comment]
Screenshot 2024-10-09 160939

File: src/web-components/SmallHandleSearch/index.ts [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+ *        </div>
+ *    </select-handle>
+ *    ```
+ */
+
+@customElement('handle-small-search')
+export class HandleSmallSearch extends LitElement {
+    @property({ type: String }) inputValue: string | undefined;
+    @property({ type: Boolean }) open: boolean = false;
+    @property({ type: String }) search: string | undefined;
```
Comment (bigirishlion) [inline src/web-components/SmallHandleSearch/index.ts:57]: Looks like this is only being used in the clearSearch. Let's remove here, in the story and jsdocs comments

**Review by bigirishlion — CHANGES_REQUESTED** [review]
🪣

**Review by bigirishlion — APPROVED** [review]
📛

**Pull request merged**